src/context.ts

```diff
@@ -492,6 +492,7 @@ class Context extends EventEmitter implements ContextEventEmitter {
   }
 
   _urlForPath(path: string, isWebSocket: boolean, options: MojoURLOptions): string {
+    path = path === '' ? '/' : path;
```

Comment:
Now the inefficiency of `path = path` bugs me a little. 😁

Reply:
mmm pushed a version (not squashed yet) with:
```js
if (path === '') {
  path = '/';
};
```
It actually seems easier to read, right?

Reply:
Although in this particular case I'm pretty sure the brackets don't help readability